[2024-09-30T17:42:13.601+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-30T17:42:13.622+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reddit_pipeline.reddit_extration manual__2024-09-30T17:42:12.251478+00:00 [queued]>
[2024-09-30T17:42:13.629+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reddit_pipeline.reddit_extration manual__2024-09-30T17:42:12.251478+00:00 [queued]>
[2024-09-30T17:42:13.630+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-09-30T17:42:13.640+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): reddit_extration> on 2024-09-30 17:42:12.251478+00:00
[2024-09-30T17:42:13.645+0000] {standard_task_runner.py:63} INFO - Started process 65 to run task
[2024-09-30T17:42:13.647+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'reddit_pipeline', 'reddit_extration', 'manual__2024-09-30T17:42:12.251478+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/reddit_dag.py', '--cfg-path', '/tmp/tmpiysk5sih']
[2024-09-30T17:42:13.649+0000] {standard_task_runner.py:91} INFO - Job 8: Subtask reddit_extration
[2024-09-30T17:42:13.688+0000] {task_command.py:426} INFO - Running <TaskInstance: reddit_pipeline.reddit_extration manual__2024-09-30T17:42:12.251478+00:00 [running]> on host d18f347f1f63
[2024-09-30T17:42:13.763+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Haziq' AIRFLOW_CTX_DAG_ID='reddit_pipeline' AIRFLOW_CTX_TASK_ID='reddit_extration' AIRFLOW_CTX_EXECUTION_DATE='2024-09-30T17:42:12.251478+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-30T17:42:12.251478+00:00'
[2024-09-30T17:42:13.764+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-30T17:42:13.777+0000] {logging_mixin.py:188} INFO - connected to reddit hello
[2024-09-30T17:42:15.005+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-30T17:42:15.006+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/pipelines/reddit_pipeline.py", line 10, in reddit_pipeline
    posts = extract_post(instance, subreddit, time_filter, limit)
  File "/opt/airflow/etls/reddit_etl.py", line 24, in extract_post
    print(post_dict.author_fullname)
AttributeError: 'dict' object has no attribute 'author_fullname'
[2024-09-30T17:42:15.019+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=reddit_pipeline, task_id=reddit_extration, run_id=manual__2024-09-30T17:42:12.251478+00:00, execution_date=20240930T174212, start_date=20240930T174213, end_date=20240930T174215
[2024-09-30T17:42:15.031+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 8 for task reddit_extration ('dict' object has no attribute 'author_fullname'; 65)
[2024-09-30T17:42:15.065+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-09-30T17:42:15.080+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-30T17:42:15.082+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
